presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -113,41 +142,126 @@ The configuration properties of Presto C++ workers are described here, in alphab
   worker node. Memory for system usage such as disk spilling and cache prefetch are
   not counted in it.
 
-``query-reserved-memory-gb``
+``shared-arbitrator.reserved-capacity``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
The formatting line (`^^^`) should have the same number of characters as the line above it.